### PR TITLE
Simplepush docs

### PR DESCRIPTION
**Pull request by fabaff**
Initial docs for Simplepush notifications.

### CODE REVIEW

File: source/_components/notify.simplepush.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+The `simplepush` platform uses [Simplepush](https://simplepush.io/) to delivery notifications from Home Assistant to your Android device.
+
+To add Simplepush to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+notify:
+  - name: NOTIFIER_NAME
+    platform: simplepush
+    device_key: ABCDEFGHJKLMNOPQRSTUVXYZ
```
Comment (Landrash) [inline source/_components/notify.simplepush.markdown]: Tested and keys I got are much shorter. Could be good to shorten it in example to avoid confusion.

Comment (fabaff) [inline source/_components/notify.simplepush.markdown]: It's a 5 chars key in my case. Agreed I will change it.

Comment (Landrash) [inline source/_components/notify.simplepush.markdown]: Same here on multiple devices.

**Comment by Landrash** [comment]
Looks good :dolphin:

**Pull request merged**